api/openapi-spec/swagger.json

```diff
@@ -16,7 +16,6 @@
   "paths": {
    "/apis/kubevirt.io": {
     "get": {
-     "description": "Get a KubeVirt API group",
```

Comment:
something strange is going on in your environment to cause this description removal in the swagger.json. you probably aren't meaning to do this.

Reply:
I think it happened because of `glide up` that updated swagger library and make generate I will revert it back

Reply:
I think it happened because of glide up that updated swagger library and make generate I will revert it back